internal/pkg/encoding/json/schema/error.go

```diff
@@ -31,8 +31,9 @@ func (v ValidationError) Error() string {
 	return v.message
 }
 
-func (v ValidationError) WriteError(w errors.Writer, _ int, _ errors.StackTrace) {
+func (v ValidationError) WriteError(w errors.Writer, level int, _ errors.StackTrace) {
 	// Disable other formatting
+	w.WriteBullet(level)
 	w.Write(v.Error())
 }
```

Comment:
`WriteError` metoda je uz zodpovedna za vypis celej chyby, predtym to bolo divne, ze odrazku vypisala libka a metoda doplnila zvysok - co bolo divne hlavne ak si chcel vypisat viacero riadkov.

Pri prvom si nemusel vypisat `WriteBullet` ale pri dalsich ano.